pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -689,11 +689,18 @@ const outputLabelConfJsonParseNoretryTemplate = `{{- define "outputLabelConfJson
 	  pod_info        ${if record['kubernetes_info'] != nil && record['kubernetes_info'] != {}; "pod_name=" + record['kubernetes_info']['pod_name']; else nil; end}
 	  container_info  ${if record['kubernetes_info'] != nil && record['kubernetes_info'] != {}; "container_name=" + record['kubernetes_info']['container_name']; else nil; end}
 	  msg_key         ${if record.has_key?('message') && record['message'] != nil; record['message']; else nil; end}
-      msg_info        ${if record['msg_key'] != nil && record['msg_key'].is_a?(Hash); require 'json'; "message="+record['message'].to_json; elsif record['msg_key'] != nil; "message="+record['message']; else nil; end}
-      message         ${if record['msg_key'] != nil && record['kubernetes_info'] != nil && record['kubernetes_info'] != {}; record['namespace_info'] + ", " + record['container_info'] + ", " + record['pod_info'] + ", " + record['msg_info']; else record['message']; end}
+	  msg_info        ${if record['msg_key'] != nil && record['msg_key'].is_a?(Hash); require 'json'; "message="+record['message'].to_json; elsif record['msg_key'] != nil; "message="+record['message']; else nil; end}
+	  message         ${if record['msg_key'] != nil && record['kubernetes_info'] != nil && record['kubernetes_info'] != {}; record['namespace_info'] + ", " + record['container_info'] + ", " + record['pod_info'] + ", " + record['msg_info']; else record['message']; end}
+	  systemd_info    ${if record.has_key?('systemd') && record['systemd']['t'].has_key?('PID'); record['systemd']['u']['SYSLOG_IDENTIFIER'] += "[" + record['systemd']['t']['PID'] + "]"; else {}; end}
 	</record>
-	remove_keys kubernetes_info, namespace_info, pod_info, container_info, msg_key, msg_info
+	remove_keys kubernetes_info, namespace_info, pod_info, container_info, msg_key, msg_info, systemd_info
```

Comment:
@vimalk78 this repeats the pattern for the other fields but why do we "add" a key in the preceding line and subsequently remove it.  Same question applies to all the keys